InnerEye/Azure/azure_config.py

```diff
@@ -122,6 +123,8 @@ class AzureConfig(GenericConfig):
     _workspace: Workspace = param.ClassSelector(class_=Workspace,
                                                 doc="The cached workspace object that has been created in the first"
                                                     "call to get_workspace")
+    model_id: str = param.String(doc="A model id string in the form 'model name:version' "
+                                     "to use a registered model for inference.")
```

Comment:
See above, this param also feels out of place here. Maybe better in WorkflowParams?

Reply:
Moved to `WorkflowParams`.